ddr/ddr/Models/GameEntities/Rider.swift

```diff
@@ -14,7 +14,9 @@ class Rider: SKSpriteNode {
   static let HALF_SCREEN_WIDTH = UIScreen.main.bounds.width / 2
```

Comment:
Thanks for re-writing this stuff with GameScene.WIDTH - we should get rid of this now